src/main/resources/hudson/tasks/Mailer/global_es.properties

```diff
@@ -20,18 +20,15 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 # THE SOFTWARE.
 
-E-mail\ Notification=Notificaci�n por correo electr�nico
+E-mail\ Notification=Notificaci�n por correo electr�nico
```

Comment:
@ndeloof Just looking at the diff on GitHub it looks like the encoding got broken. Could you double-check this (this and other properties files)?